README.md

```diff
@@ -163,6 +163,17 @@ console.log(profile.pubkey)
 // prints: 32e1827635450ebb3c5a7d12c1f8e7b2b514439ac10a67eef3d9fd9c5c68e245
 console.log(profile.relays)
 // prints: [wss://relay.damus.io]
+
+// or alternative for nodejs
+import 'websocket-polyfill'
```

Comment:
what browsers don't support websockets?

Reply:
It's for nodejs. Besides, it's recommended to do it in the readme.md on the main page.